docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.5'
+services:
+
+  db:
+    container_name: ctech-db
+    image: mongo:4.0.13
+    
+  web:  
+    container_name: ctech-web
+    image: node:13
+    working_dir: /comunidadestech
+    env_file: .env
+    depends_on: 
+      - db
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/comunidadestech
+      - yarn_cache:/comunidadestech/node_modules/
```

Comment:
Não sei se é o ideal mantermos a pasta node_modules como volume, talvez o ideal fosse até colocar em um arquivo .dockerignore para que não seja utilizado quando for dado build na aplicação